modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -190,6 +190,48 @@ autoSaveAsDraftTest(
 	}
 );
 
+autoSaveAsDraftTest(
+	'LPD-26863: Undo/Redo buttons work with metadata fields',
+	async ({journalEditArticlePage, page, site}) => {
+		const changesSavedIndicator = await page.locator(
+			'#_com_liferay_journal_web_portlet_JournalPortlet_changesSavedIndicator'
+		);
+		const redoButton = page.getByTitle('Redo', {exact: true});
+		const title = getRandomString();
+		const undobutton = page.getByTitle('Undo', {exact: true});
+
+		await journalEditArticlePage.goto({siteUrl: site.friendlyUrlPath});
+
+		await journalEditArticlePage.titlePlaceholder.click();
```

Comment:
Yeah I tried it, but it does not work because when we click and then fill the title, the inputs does not update, I think this issue is because of teh delays of Liferay.fire events, that's why I added a timeout after the click